switcheroo/ssh/scripts/publish.py

```diff
@@ -37,22 +42,50 @@ def create_argument_parser() -> ArgumentParser:
             the directory that stores local keys (ie /home/you/.ssh)",
         default=paths.local_ssh_home(),
     )
+    argument_parser.add_argument(
+        "-m",
+        "--metric",
+        action="store_true",
+        choices=["file", "cloud"],
```

Comment:
Maybe rename cloud to aws - allows for future cloud implementation (ie if we add google cloud, can put google)